packages/adblocker-benchmarks/package.json

```diff
@@ -63,5 +63,9 @@
       "name": "Anton Lazarev",
       "email": "antonok35@gmail.com"
     }
-  ]
+  ],
+  "devDependencies": {
+    "rollup": "^2.55.1",
+    "rollup-plugin-terser": "^7.0.2"
+  }
```

Comment:
Do we need this if uBlock Origin already ships as a bundle?

Reply:
Sorry about the confusion: It doesn't ship with the bundle, but the bundle is built on the client system by the install script. I had done it that way for expedience but I realize it's not necessary. Moreover, that method creates a dependence on external tools within the ubo-snfe package itself. I'll clean that up over there. I think @gorhill  is not a fan of the dependencies either.

Reply:
Also see https://github.com/cliqz-oss/adblocker/pull/2095#discussion_r680715519